frontend/src/main/java/puf/frisbee/frontend/view/TopPanelView.java

```diff
@@ -11,80 +11,180 @@
 import puf.frisbee.frontend.viewmodel.*;
 
 public class TopPanelView {
+    /**
+     * The team data to display in the top panel in fxml.
+     */
     @FXML
     private HBox teamDataTopPanel;
 
+    /**
+     * The top panel to display in the start view in fxml.
+     */
```

Comment:
Jetzt weiss ich wieder, worum es in dem Aufräumticket ging...  das waiting view top panel und das game view top panel sind komplett identisch (im fxml), da könnte man einfach eins nehmen und sich die Hälfte aller labels sparen 😅